gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -4518,7 +4518,7 @@ gnc_plugin_page_register_cmd_delete_transaction (GtkAction* action,
 }
 
 static void
-gnc_plugin_page_register_cmd_associate_transaction (GtkAction *action,
+gnc_plugin_page_register_cmd_linked_transaction (GtkAction *action,
                                                     GncPluginPageRegister* plugin_page)
```

Comment:
Complete nitpick: the function name is shorter now, but the function parameters on the second line are still aligned on the old function name length. I only saw this because you fixed it for a single instance a bit further down this file.